src/components/Page3.jsx

```
@@ -1,14 +1,78 @@
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import LoginPagesInspector from "./UI/LoginPagesInspecter";
+import axios from "axios";
 
-const Page2 = () => {
+const Page3 = () => {
   const [currentPage, setCurrentPage] = useState(3); // Add state for current page
-
+  const [pharmacyName, setPharmacyName] = useState("");
+  const [pharmacyAddress, setPharmacyAddress] = useState("");
+  const [email, setEmail] = useState(""); // Add email state
```

ESLint

[error] 'email' is assigned a value but never used.

(no-unused-vars)